test-suite/doublebarrieroption.cpp

```diff
@@ -562,14 +534,9 @@ void DoubleBarrierOptionTest::testVannaVolgaDoubleBarrierValues() {
     }
 }
 
-test_suite* DoubleBarrierOptionTest::suite(SpeedLevel speed) {
```

Comment:
Ehm, did you just copy this file from another branch over the existing one?  This change breaks the build because the interface of the method doesn't match.

The same goes for the code you deleted above: it looks like you overwrote changes that were made since you made your branch.